contracts/StablecoinConverter.sol

```diff
@@ -133,6 +142,7 @@ contract StablecoinConverter is EpochTokenLocker {
         uint128[] memory volumes,
         uint128[] memory prices,  //list of prices for touched token only
         uint16[] memory tokenIdsForPrice  // price[i] is the price for the token with tokenID tokenIdsForPrice[i]
+                                          // tokenId for fee token does not need to be send, as it is always 0
```

Comment:
I'm not sure if the added complexity here (shifting binary search by one, requiring 0 to not be part of the tokenId list) is worth the one uint16 calldata saving. I'd instead just assert that the first entry is 0 and keep the remaining logic as is.

Reply:
Yeah, I also did not implement that immediately. But since we are also storing the tokenIdsForPrice, I thought that is worth the complexity. But for sure, you have a point here.

Reply:
Overall there are too many (5) confusing small changes that are all related to saving a tiny amount of data here (that will also prevent us from using standard library implementations).